### ISSUE TITLE
--realpath option on artisan migrate [L5.5] does not exist

### ISSUE THREAD

**Issue by andrewmclagan** [orchestral/testbench#213]
There are allot of issues stating this with a 

> "Read the docs"

comment.

After investigation and installing version `3.5.x` of this package corresponding to laravel `5.5.x` and looking at the internals of the [`Illuminate\Database\Console\Migrations\Migrate`](https://github.com/laravel/framework/blob/5.6/src/Illuminate/Database/Console/Migrations/MigrateCommand.php#L17) command. There really is no `--realpath` option for `artisan migrate`.

From [`Illuminate\Database\Console\Migrations\Migrate`](https://github.com/laravel/framework/blob/5.6/src/Illuminate/Database/Console/Migrations/MigrateCommand.php#L17)

```php
    protected $signature = 'migrate {--database= : The database connection to use.}
                {--force : Force the operation to run when in production.}
                {--path= : The path of migrations files to be executed.}
                {--pretend : Dump the SQL queries that would be run.}
                {--seed : Indicates if the seed task should be re-run.}
                {--step : Force the migrations to be run so they can be rolled back individually.}';
```

and from [`Orchestra\Testbench\Concerns\WithLoadMigrationsFrom`](https://github.com/orchestral/testbench-core/blob/3.5/src/Concerns/WithLoadMigrationsFrom.php#L24)

```php
        $options = is_array($realpath) ? $realpath : ['--realpath' => $realpath];

        $this->artisan('migrate', $options);
```

Unless of course you are overriding this command, which I doubt.

**Comment by andrewmclagan** [comment]
Whooops you have a typo in your docs namespace on `3.5.x`

`Orchestra\Database\ConsoleServiceProvider\ConsoleServiceProvider::class,`

should be 

`Orchestra\Database\ConsoleServiceProvider::class,`

**Comment by andrewmclagan** [comment]
Wrote up a PR 

https://github.com/orchestral/testbench/pull/214

**Comment by jshayes** [comment]
I just ran into this issue. Seems like the `Orchestra\Testbench\Concerns\WithLoadMigrationsFrom` trait in version `3.5.x` does not work with Laravel `5.5`. From what I can see, `--realpath` was [introduced in Laravel `5.6`](https://github.com/laravel/framework/pull/22852)

**Comment by andrewmclagan** [comment]
@jshayes No it works. Read this issue and the related PR. All the info is there and in the documentation.

**Comment by jshayes** [comment]
@andrewmclagan Ah, you're right. My test case was missing the `getPackageProviders` method, adding it fixed the problem.

Thanks.

**Comment by andrewmclagan** [comment]
Yeah i think it needs better documenting.

**Comment by crynobone** [comment]
> Yeah i think it needs better documenting. I had the same issue.

Not worth it, `--realpath` is available with Laravel 5.6 and therefore there no reason to have `orchestra/database` as you would need in earlier versions.

**Comment by spawnia** [comment]
This describes how to resolve the issue when using Laravel 5.5:

https://github.com/orchestral/testbench/tree/3.5#running-testing-migrations
